packages/slice-machine/pages/index.tsx

```diff
@@ -181,11 +182,11 @@ const CustomTypes: React.FunctionComponent = () => {
           }
         />
       ) : (
-        <Grid
+        <Grid // The CtPayload should be used here once the Context has been changed.
           elems={customTypes}
-          renderElem={(ct: CtPayload) => (
+          renderElem={(ct: CustomType<ObjectTabs>) => (
             <Link passHref href={`/cts/${ct.id}`} key={ct.id}>
-              <Card ct={ct} />
+              <Card ct={ct as CtPayload} />
```

Comment:
weird, how come the type had to be cast here?

Reply:
hey, I put a comment about that a couple of lines before.
The Grid components takes elements and then a function to transform an element.
The entry of this Grid element was `CustomType<ObjectTabs>` but in the function to render the element itself it was casted as `CtPayload`.
Since I typed the Grid Components, we now know that ct is a `CustomType<ObjectTabs>` but I had to cast it for Card to work, not knowing which of the two types was suppose to be the right one.